Source/NSToolbar.m

```diff
@@ -583,39 +583,41 @@ - (id) initWithCoder: (NSCoder *)aCoder
     {
       ASSIGN(_identifier, [aCoder decodeObjectForKey:@"NSToolbarIdentifier"]);
       _items = [[NSMutableArray alloc] init];
-  
+
       ASSIGN(_interfaceBuilderItemsByIdentifier, [aCoder decodeObjectForKey: @"NSToolbarIBIdentifiedItems"]);
       ASSIGN(_interfaceBuilderAllowedItemIdentifiers, [self _identifiersForItems: [aCoder decodeObjectForKey: @"NSToolbarIBAllowedItems"]]);
       ASSIGN(_interfaceBuilderDefaultItemIdentifiers, [self _identifiersForItems: [aCoder decodeObjectForKey: @"NSToolbarIBDefaultItems"]]);
       ASSIGN(_interfaceBuilderSelectableItemIdentifiers, [self _identifiersForItems: [aCoder decodeObjectForKey: @"NSToolbarIBSelectableItems"]]);
-      
+
       _customizationPaletteIsRunning = NO;
       _configurationDictionary = nil;
 
       [self setAllowsUserCustomization: [aCoder decodeBoolForKey: @"NSToolbarAllowsUserCustomization"]];
       [self setAutosavesConfiguration: [aCoder decodeBoolForKey: @"NSToolbarAutosavesConfiguration"]];
-      [self setDisplayMode: [aCoder decodeIntForKey: @"NSToolbarDisplayMode"]]; 
+      [self setDisplayMode: [aCoder decodeIntForKey: @"NSToolbarDisplayMode"]];
       [self setShowsBaselineSeparator: [aCoder decodeBoolForKey: @"NSToolbarShowsBaselineSeparator"]];
       [self setSizeMode: [aCoder decodeIntForKey: @"NSToolbarSizeMode"]];
       [self setVisible: [aCoder decodeBoolForKey: @"NSToolbarPrefersToBeShown"]];
       [self setDelegate: [aCoder decodeObjectForKey: @"NSToolbarDelegate"]];
-    } 
+    }
   else
     {
       // FIXME
     }
 
+  [self _build];
```

Comment:
A huge -- although necessary -- cleanup diff obscures the fact that  `[self _build];` is the only code change.

How does this affect `NSToolbar` behavior? Why did it work before?

Reply:
I am again sorry, this is another result of delete-training-whitespace. My only explanation why this worked before is that most likely we never had a toolbar encoded.

Reply:
>  this is another result of delete-training-whitespace. 

Sorry again for not noticing your direct note that I should ignore the whitespace, my mistake!